src/utils/jellyfin-apiclient/getItemsHelper.ts

```diff
@@ -0,0 +1,63 @@
+import type {BaseItemDtoQueryResult} from '@jellyfin/sdk/lib/generated-client';
```

Comment:
I think this file can just be named `getItems.ts`... "Helper" seems a bit redundant. :slightly_smiling_face:

Reply:
That's fine with me! I'll rename it. Is the quoted line just to be able to make a comment? I am confused about that.